### PR TITLE
Adding healthcheck endpoint and also fixed datadog metric submission

### PR DESCRIPTION
**Pull request by yiwork**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,12 +13,35 @@ services:
...
+    volumes:
+      - $PWD:/app
+    env_file:
+      - ".env"
+    depends_on:
+      - localpostgres
+    profiles:
+      - dev
+
+  localpostgres:
```
Comment (mattcasey) [inline docker-compose.yml:36]: so we now have it set up to run locally? neat - I'd like to add the commands to our README. would you mind? or tell me what would the commands be to start up docker

File: pages/api/health/index.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import log from 'lib/log';
+import { gauge, count } from 'lib/metrics';
+
+export default function handler (req: NextApiRequest, res: NextApiResponse) {
+  res.status(200).json({ health: 'ok' });
+}
+
+// need to add a router to this.
+export function testDDGauge (req: NextApiRequest, res: NextApiResponse) {
+  gauge('.health.dd.gauge', 99);
```
Comment (mattcasey) [inline pages/api/health/index.ts:11]: i dont think you need the starting "." right?

**Review by mattcasey — APPROVED** [review]
approved with one question

File: pages/api/health/index.ts (hunk 2 above)
Comment (mattcasey) [reply pages/api/health/index.ts:11]: I noticed it looks right in datadog, maybe they do something special with the prefix? Do you think we always need to have a "."

File: docker-compose.yml (hunk 1 above)
Comment (yiwork) [reply docker-compose.yml:36]: We can have it setup to run locally if we chose to. Though I haven't actually ran it to see if it worked. Can work on that next.

File: pages/api/health/index.ts (hunk 2 above)
Comment (yiwork) [reply pages/api/health/index.ts:11]: Good question, don't know. I'll remove it and see what happens

**Pull request merged**